### PR TITLE
New version of FDTD_Solutions  with patches for hardcoded paths etc.

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
New FDTD_Solutions v8.16.982

### CODE REVIEW

File: easybuild/easyconfigs/f/FDTD_Solutions/FDTD_Solutions-8.16-fix-interconnect-api-path.patch [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+diff -ru opt.orig/lumerical/fdtd/api/python/lumapi.py opt/lumerical/fdtd/api/python/lumapi.py
+--- opt.orig/lumerical/fdtd/api/python/lumapi.py	2016-09-10 02:54:25.000000000 +0200
++++ opt/lumerical/fdtd/api/python/lumapi.py	2017-02-02 12:54:37.413273068 +0100
+@@ -1,7 +1,8 @@
+ from ctypes import *
+ from numpy import *
++import os
+ 
+-INTEROPLIB = "/opt/lumerical/interconnect/api/c/libinterop-api.so.1"
++INTEROPLIB = os.environ['EBROOTINTERCONNECT'] + "/api/c/libinterop-api.so.1"
```
Comment (boegel) [inline easybuild/easyconfigs/f/FDTD_Solutions/FDTD_Solutions-8.16-fix-interconnect-api-path.patch]: Where/how is `$EBROOTINTERCONNECT` supposed to get defined?

Shouldn't this be `$EBROOTFDTD_SOLUTIONS/interconnect`?

Comment (akesandgren) [reply easybuild/easyconfigs/f/FDTD_Solutions/FDTD_Solutions-8.16-fix-interconnect-api-path.patch]: I originally thought it was supposed to point to Lumericals INTERCONNECT installation and its libinterop-api.so. But having looked at the corresponding file in INTERCONNECT and the fact that libinterop-api exists in fdtd_solutions too makes things a bit clearer.

Patch upcoming.

**Comment by akesandgren** [comment]
@boegel ping?

**Comment by boegel** [comment]
(close/re-open to trigger updated Travis tests)

@akesandgren Looks like `checksums` will need to be added.

A test report would also be nice since the sources are not freely available (?).

Or do we close this as is, superseded by #7088?

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/b86527b01b7db21ac8353cbb7ff1f608 for a full test report.

**Comment by akesandgren** [comment]
Please merge this instead of closing.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**